packages/ivass-certified-attributes-importer/src/model/attributeModel.ts

```diff
@@ -0,0 +1,8 @@
+import { z } from "zod";
+
+export const PersistentAttribute = z.object({
+  id: z.string().uuid(),
+  origin: z.string(),
+  code: z.string(),
+});
+export type PersistentAttribute = z.infer<typeof PersistentAttribute>;
```

Comment:
Same comment from other PR, for this you can probably reuse the `Attribute` model